gemini_test.py

```diff
@@ -67,6 +67,38 @@ def test_load_random_with_nemesis(self):
 
         self.verify_results()
 
+    def test_load_random_with_nemesis_cdc_reader(self):
+        cmd = self.params.get('gemini_cmd')
+        cdc_stress = self.params.get('stress_cdclog_reader_cmd')
+        update_es = self.params.get('store_cdclog_reader_stats_in_es')
+
+        self.db_cluster.add_nemesis(nemesis=self.get_nemesis_class(),
+                                    tester_obj=self)
+
+        self.log.debug('Start gemini benchmark')
+        gemini_thread = self.run_gemini(cmd=cmd)
+        self.gemini_results["cmd"] = gemini_thread.gemini_commands
+        # waite gemini create schema
+        time.sleep(15)
```

Comment:
Why not waiting for schema agreement or  something more deterministic if possible?

Reply:
Changed to wait schema agreement.